packages/lesswrong/components/messaging/InboxNavigation.tsx

```diff
@@ -7,11 +7,11 @@ import qs from 'qs'
 import { forumTypeSetting } from '../../lib/instanceSettings';
 
 // The Navigation for the Inbox components
-const InboxNavigation = ({classes, terms, currentUser, title="Your Conversations"}: {
+const InboxNavigation = ({classes, terms, currentUser, title=<>Your Conversations</>}: {
   classes: ClassesType,
   terms: ConversationsViewTerms,
   currentUser: UsersCurrent,
-  title?: String
+  title?: JSX.Element
```

Comment:
This is mis-typed, if a string can't be passed in. You could do JSX.Element | String if you wanted, but I think there's a better one.